### PR TITLE
Added Basic UI

### PR DESCRIPTION
**Pull request by JacintaNgwu**
- Built an interactive console apps.

- Created an app.rb file that serves as a console app entry-point. It contains methods that do the following:
List all books.
List all people.
Create a person (teacher or student, not a plain Person).
Create a book.
Create a rental.
List all rentals for a given person id.

- Created main.rb file that defines the entry point, that is invoked at the end of the file. This method do the following:
Present the user with a list of options to perform.
Lets users choose an option.
If needed, ask for parameters for the option.
Have a way to quit the app.

**Below is a screenshot of the console**
image
image

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    id = gets.chomp.to_i
+
+    rentals = @rentals.filter { |rental| rental.person.id == id }
+    puts 'List of Rentals:'
+    rentals.each do |rental|
+      puts "Date: #{rental.date}, Book '#{rental.book.title}' by #{rental.book.author}"
+    end
+    display_list
+    continue
+  end
```
Comment (AmaduKamara) [inline app.rb]: - [x] When I try to display a rental for a person, the app crashes with the error in the screenshot below. Kindly fix that 👍🏽

Screenshot 2023-03-17 at 12 06 22 PM

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    when '4' then create_book
+    when '5' then create_rental
+    when '6' then list_rentals_of_person_id
+    when '7'
+      colorize_output(34, 'Thank you for using this application!!')
+      exit
+    end
+  end
+  # rubocop:enable Metrics/CyclomaticComplexity
+end
```
Comment (AmaduKamara) [inline app.rb]: - [ ] Let's try not to disable Linter checks as they help us write clean and professional code. Disabling linter checks defies its purpose. Kindly fix the Linter error 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes  ♻️ ♻️ ♻️

Hi @JacintaNgwu,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights:

✅ You used Gityflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Good descriptive PR 👍🏽

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @JacintaNgwu, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**